src/components/PageHeading/index.scss

```diff
@@ -0,0 +1,5 @@
+.easi-h1 {
+  &:focus {
+    outline: none !important;
```

Comment:
`tabIndex={-1}`allows an element to be focused through Javascript, but **not** via tabs.

The `outline` is a focus indicator for users using keyboard controls, allowing them to visually see where they are on the page.

Since the `h1` is a non-interactive element AND it can only be focused via Javascript (not keyboard controls), IMO, I don't think it's necessary to have the outline. It can be distracting for screen users.